File: Programming Basics with Python/While-цикъл/password.py

username = input()
password = input()

input_pass = input()
while password != input_pass:
    input_pass = input()
    
print(f"Welcome {username}!")
    

"""
username = input()
password = input()

while True:
    input_pass = input()
    if password == input_pass:
        print(f"Welcome {username}!")
        break
"""

File: Programming Basics with Java/While-loop/Password.java
import java.util.Scanner;

public class Password 
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);

        String name = input.nextLine();
        String password = input.nextLine();
        String pass = input.nextLine();

        while (!pass.equals(password))
        {
            pass = input.nextLine();
        }

        System.out.printf("Welcome %s!", name);
    }
}
